central/signatureintegration/datastore/datastore_impl.go

```diff
@@ -135,6 +144,16 @@ func (d *datastoreImpl) RemoveSignatureIntegration(ctx context.Context, id strin
 		return err
 	}
 
+	// Get the integration to check its origin
+	integration, err := d.getSignatureIntegrationByID(ctx, id)
```

Comment:
This seems to make the above `verifyIntegrationIDExists` call redundant. Could you please check if this can be removed now?

Reply:
Indeed, and I [had actually removed that](https://github.com/stackrox/stackrox/pull/16478#discussion_r2287777450) in the draft version, somehow lost the change. Done: https://github.com/stackrox/stackrox/pull/16478/commits/23a8cf8eadf38416ea06e3835abfa921115c82f8